File: app/src/main/java/com/jobtick/android/models/payments/Task.java
package com.jobtick.android.models.payments;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Task implements Serializable {
    @SerializedName("id")
    @Expose
    private Integer id;
    @SerializedName("title")
    @Expose
    private String title;
    @SerializedName("slug")
    @Expose
    private String slug;
    @SerializedName("closed_at")
    @Expose
    private Object closedAt;
    @SerializedName("completed_at")
    @Expose
    private Object completedAt;
    @SerializedName("created_at")
    @Expose
    private String createdAt;
    @SerializedName("poster")
    @Expose
    private Poster poster;
    @SerializedName("worker")
    @Expose
    private Worker worker;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getSlug() {
        return slug;
    }

    public void setSlug(String slug) {
        this.slug = slug;
    }

    public Object getClosedAt() {
        return closedAt;
    }

    public void setClosedAt(Object closedAt) {
        this.closedAt = closedAt;
    }

    public Object getCompletedAt() {
        return completedAt;
    }

    public void setCompletedAt(Object completedAt) {
        this.completedAt = completedAt;
    }

    public String getCreatedAt() {
        return createdAt;
    }

    public void setCreatedAt(String createdAt) {
        this.createdAt = createdAt;
    }

    public Poster getPoster() {
        return poster;
    }

    public void setPoster(Poster poster) {
        this.poster = poster;
    }

    public Worker getWorker() {
        return worker;
    }

    public void setWorker(Worker worker) {
        this.worker = worker;
    }

}

File: app/src/main/java/com/jobtick/android/models/response/conversationinfo/Task.kt
package com.jobtick.android.models.response.conversationinfo

import com.google.gson.annotations.SerializedName

data class Task(
    @SerializedName("amount") val amount: Int?,
    @SerializedName("closed_at") val closed_at: Any?,
    @SerializedName("completed_at") val completed_at: Any?,
    @SerializedName("created_at") val created_at: String?,
    @SerializedName("due_date") val due_date: String?,
    @SerializedName("id") val id: Int?,
    @SerializedName("slug") val slug: String?,
    @SerializedName("status") val status: String?,
    @SerializedName("task_type") val task_type: String?,
    @SerializedName("title") val title: String?
)